LondonBritishLibrary/orient/BLorient611.xml

```diff
@@ -0,0 +1,249 @@
+<title type="complete" ref="LIT1295Dersan#HomilyTerr"/>
+<textLang xml:lang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.4">
+<locus from="25v"/>
+<title type="complete" ref="LIT1295Dersan#HomilyYakkatit"/>
+<textLang xml:lang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.5">
+<locus from="29r"/>
+<title type="complete" ref="LIT1295Dersan#HomilyMaggabit"/>
+<textLang xml:lang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.6">
+<locus from="43v"/>
+<title type="complete" ref="LIT1295Dersan#HomilyMiyazya"/>
+<textLang xml:lang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.7">
+<locus from="48r"/>
+<title type="complete" ref="LIT1295Dersan#HomilyGenbot"/>
+<textLang xml:lang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.8">
+<locus from="50r"/>
+<title type="complete" ref="LIT1295Dersan#HomilySane"/>
+<textLang xml:lang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.9">
+<locus from="56r"/>
+<title type="complete" ref="LIT1295Dersan#HomilyHamle"/>
+<textLang xml:lang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.10">
+<locus from="60r"/>
+<title type="complete" ref="LIT1295Dersan#HomilyNahase"/>
+<textLang xml:lang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.11">
+<locus from="63r"/>
+<title type="complete" ref="LIT1295Dersan#HomilyMaskaram"/>
+<textLang xml:lang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.12">
+<locus from="66r"/>
+<title type="complete" ref="LIT1295Dersan#HomilyTeqemt"/>
+<textLang xml:lang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.13">
+<locus from="70r"/>
+<title type="complete" xml:lang="en">The miracles</title>
+<textLang xml:lang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.14">
+<locus from="85r"/>
+<title type="incomplete" xml:lang="en" ref="LIT2375Synaxa">Extracts from the Synaxarium regarding the festivals</title>
+<textLang xml:lang="gez"></textLang>
+</msItem>
+
+</msItem>
+
+</msContents>
+
+<physDesc>
+<objectDesc form="Codex">
+  <supportDesc>
+      <support>
+          <material key="parchment"/>
+      </support>
+      <extent>
+          <measure unit="leaf">92</measure>
+          <measure unit="blank">Two unnumbered blank leaves at the end.</measure>
+          <dimensions type="outer" unit="in">
+              <height>11.5</height>
+              <width>9.625</width>
+              </dimensions>
+            </extent>
+    </supportDesc>
+
+    <layoutDesc>
+        <layout columns="2" writtenLines="19 23" cert="high">
+        </layout>
+    </layoutDesc>
+</objectDesc>
+
+<handDesc>
+
+    <handNote script="Ethiopic" xml:id="h1">
+    <locus from="5" to="84"/>
+    <date notBefore="1700" notAfter="1799" resp="PRS10171Wright"/>
+    </handNote>
+
+    <handNote script="Ethiopic" xml:id="h2">
+    <locus from="85" to="92"/>
+    <date notBefore="1800" notAfter="1899" resp="PRS10171Wright"/>
+    </handNote>
+
+</handDesc>
+
+<decoDesc>
+<summary></summary>
+<decoNote type="miniature" xml:id="d1">
```

Comment:
sorry, you would need to create two decoNotes for these miniatures

Reply:
Sure, still it would be nice to know what's on them :/ I moved the info to summary

Reply:
And why do you need to put information into a summary? It stays now the very same information that you have encoded in decoNotes. What do you want to point out?

Reply:
I understand that summary is not needed here, since we have the same info below.